.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,36 @@
-- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+<!--
+Thank you for sending the PR! 
+Please fill the applicable details below
+Happy contributing!
+-->
 
-- **Why was this change needed?** (You can also link to an open issue here)
+### What kind of change does this PR introduce? 
+<!-- Mark All The Applicable Boxes -->
 
-- **Other information**:
+- [ ] Bug
+- [ ] Feature
+- [ ] Docs
+- [ ] Other(s)
+
+
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
+### Fixes Issue
```

Comment:
I think this was not included in the old template? 
There is not not always an issue so I would rather not see this heading at all, could be included in "Why was this change needed"
@ainouzgali @p-fernandez what do you think?

Reply:
Let me know what others think about it, and I'll remove if more agree.

Reply:
@MrKrishnaAgarwal please follow @davidsoderberg suggestion.